keps/sig-node/5067-pod-generation/README.md

```diff
@@ -625,16 +635,16 @@ in back-to-back releases.
 * `metadata.generation` functionality implemented
 * `status.observedGeneration` functionality implemented behind feature flag
 * `status.conditions[i].observedGeneration` field added to the API
+* `status.conditions[i].observedGeneration` functionality implemented behind feature flag
 
 #### Beta
 
 * `metadata.generation`,  `status.observedGeneration`,  `status.conditions[i].observedGeneration` functionality have been implemented and running as alpha for at least one release
-* `status.conditions[i].observedGeneration` functionality implemented behind feature flag
 
 #### GA
```

Comment:
I wonder if some of the tests must be converted to the Conformance. I think this API should be a part of conformance suite. Please add this to graduation criteria for GA.

Ideally those tests that you want to promote to Conformance must be written in beta so they run for a while as green before being promoted

Reply:
added to the GA critieria that the main e2e tests should be converted to Conformance